modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step08_abtr.md

```diff
@@ -0,0 +1,283 @@
+- [!param](/Mesh/AdvancedExtruderGenerator/bottom_boundary) and [!param](/Mesh/AdvancedExtruderGenerator/top_boundary) parameters are used to assign the boundary IDs of the bottom and top boundary sidesets
+
+### Example
+
+!media tutorial04_meshing/hom_abtr_3d_core.png
+       id=tutorial04-hom_abtr_3d_core
+       caption=3D core after extrusion, colored by subdomain IDs.
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing reactor_examples/abtr/abtr.i
+         id=tutorial04-abtr_mesh_extrude
+         caption=Homogeneous extrude example.
+         block=Mesh/extrude
+
+## Assign Plane-level Reporting IDs
+
+In order to facilitate output processing, we assign `plane_id` reporting IDs to indicate which elements on the 3D mesh belong to the same axial plane.
+
+### Object
+
+- [PlaneIDMeshGenerator.md]
+
+### Geometry Features
+
+- Assign coordinates demarking axial levels in plane_coordinates. These levels should be consistent with how axial levels were defined in [AdvancedExtruderGenerator.md].
+
+### Example
+
+!media tutorial04_meshing/hom_abtr_3d_core_with_plane_id.png
+       id=tutorial04-hom_abtr_3d_core_with_plane_id
+       caption=3D ABTR colored by "plane_id" reporting ID.
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing reactor_examples/abtr/abtr.i
+         id=tutorial04-abtr_mesh_plane_id
+         caption=Homogeneous plane ID example.
+         block=Mesh/plane_id
+
+## (Optional) Rename Outer Boundary Sidesets
+
+Since [AdvancedExtruderGenerator.md] requires top and bottom boundary sidesets be defined using numeric IDs, we can assign a name (string) such as `'top'` and `'bottom'` to these sidesets for easier reference in physics applications.
+
+### Object
+
+- [RenameBoundaryGenerator.md]
+
+### Notes
+
+- Griffin requires the outer boundary sidesets to be defined to apply boundary conditions such as vacuum or reflective boundary conditions
+
+### Example
+
+!media tutorial04_meshing/hom_abtr_3d_sidesets.png
+       id=tutorial04-hom_abtr_3d_sidesets
+       caption=Sideset names and locations of the ABTR core mesh.
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing reactor_examples/abtr/abtr.i
+         id=tutorial04-abtr_mesh_plane_rename
+         caption=Homogeneous boundary rename example.
+         block=Mesh/abtr_mesh
+
+## Use of ABTR Mesh in Downstream Physics Code (Griffin)
+
+The Reactor Module creates meshes containing blocks of elements (identified by block ID), groups of elements with similar extra element integers (identified by different extra element IDs such as `pin_id`, `assembly_id`, `depletion_id`), and groups of curves (2D meshes) or faces (3D meshes) called sidesets (identified by sideset ID). In particular, the blocks and sidesets are used in downstream physics codes to assign materials to mesh elements and to assign boundary conditions.
```

Comment:
Change to

The Reactor Module creates meshes containing blocks of elements (identified by block ID), groups of elements with similar reporting IDs (identified by different reporting IDs such as `pin_id`, `assembly_id`, `depletion_id`), and groups of curves (2D meshes) or faces (3D meshes) called sidesets (identified by sideset ID). In particular, the blocks and sidesets are used in downstream physics codes to assign materials to mesh elements and to assign boundary conditions.